messaging/methods/media/listMedia.md

```diff
@@ -81,7 +81,7 @@ var a = b;
 {% sample lang="python" %}
 
 ```python
-# Python Example
+media = messaging_client.list_media("12345")
```

Comment:
Can you add example on iterating over this object? 

Also noticed we bounce between `MESSAGING_ACCOUNT_ID` and `12345`. I personally like the `MESSAGING_ACCOUNT_ID` var over the string of numbers.

Reply:
Hmm I can, I'll have to do some digging around to get that.

Also will update to use the global var